cloudformation/alarms.yaml

```diff
@@ -5,11 +5,10 @@
 
 Resources:
   DDBToESErrorAlarm:
-    DependsOn: DdbToEsLambdaFunction

```

Comment:
Why remove this?

Reply:
[One of the dimensions in this alarm does a `!Ref DdbToEsLambdaFunction`](https://github.com/awslabs/fhir-works-on-aws-deployment/blob/88c9574338f48dc0a7241de5e5f6856061335cac/cloudformation/alarms.yaml#L19). The dependency is made that way. Was just one of the linter warnings that cfn-lint mentioned so I removed it